sig-rktnetes/README.md

```diff
@@ -0,0 +1,3 @@
+[Meeting Notes](https://docs.google.com/document/d/1otDQ2LSubtBUaDfdM8ZcSdWkqRHup4Hqt1VX1jSxh6A/edit?usp=sharing)
```

Comment:
Meeting notes should be available for anyone. At this moment i don't have permissions